cards-template1a.js

```diff
@@ -2,19 +2,39 @@
 
 // function()--> possible return values
 
```

Comment:
Looks good! Here's a tip for simplification: Instead of creating a variable that contains your equation and then returning that variable, you can drop the variable (rankOfCard, for example) and just return the equation as it is.